.github/workflows/build.yml

```diff
@@ -14,18 +14,14 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        php: [7.2, 7.3, 7.4, 8.0, 8.1, 8.2, 8.3, 8.4]
+        php: [8.1, 8.2, 8.3, 8.4]
         composer-flags: [ "" ]
         symfony-version: [ "" ]
         include:
-          -   php: 7.2
-              symfony-version: '3.*'
-          -   php: 7.3
-              symfony-version: '4.*'
-          -   php: 7.4
-              symfony-version: '5.*'
-          -   php: 8.0
-              symfony-version: '5.*'
+          -   php: 8.1
+              symfony-version: '5.4'
+          -   php: 8.1
+              symfony-version: '6.4'
```

Comment:
the step below expects `symfony-version` to be a version constraint. Using `6.4` here will make it install 6.4.0, which is not what we want

Reply:
Thanks @stof, good catch. I'd not registered the install here is different to the one in behat.